components/RichText/Quote.module.scss

```diff
@@ -1,47 +1,43 @@
 .container {
     @include border-radius-m;
 
+    display: flex;
     margin: $spacing-6 auto;
-    padding: $spacing-4;
-    background: $color-base-50;
-}
-
-.left,
-.right {
-    flex-shrink: 0;
-    width: 2px;
-    background-color: $color-base-400;
-}
-
-.left {
-    margin-right: $spacing-3;
-}
-
-.right {
-    margin-left: $spacing-3;
 }
 
 .content {
     @include paragraph;
 
-    flex: 1;
-    text-align: center;
+    padding: $spacing-4;
+    background: $color-base-50;
 }
 
 .alignLeft {
     text-align: left;
-    border-left: 2px solid $color-base-400;
-    padding-left: $spacing-3;
+    justify-content: flex-start;
+
+    .contentInner {
+        padding-left: $spacing-3;
+        border-left: 2px solid $color-base-400;
+    }
 }
 
 .alignCenter {
     text-align: center;
-    border-left: 2px solid $color-base-400;
-    padding-left: $spacing-3;
+    justify-content: center;
+
+    .contentInner {
+        padding-left: $spacing-3;
+        border-left: 2px solid $color-base-400;
+    }
 }
 
 .alignRight {
     text-align: right;
-    border-right: 2px solid $color-base-400;
-    padding-right: $spacing-3;
+    justify-content: flex-end;
+
+    .contentInner {
+        padding-right: $spacing-3;
+        border-right: 2px solid $color-base-400;
+    }
```

Comment:
I wonder if this can be solved with `:before`?

I just don't feel comfortable seeing too many wrapper divs for a simple use-case like this :)